designs/static-capacity.md

```diff
@@ -0,0 +1,236 @@
+However, the nodes provisioned by a static pool:
+- Participate in scheduling decisions for both launched capacity and yet to launch inflight capacity (i.e., pods can land on them)
+- Are monitored for drift, enabling graceful re-creation when necessary
+- Will be evenly distributed across AZs when requirements span multiple zones
+
+This means they remain first-class citizens in the cluster but are excluded from cost-based disruption decisions.
+
+## Requirements
+
+The requirements field in a static NodePool behaves identically to dynamic pools—it defines the constraints for all NodeClaims launched under that NodePool.
+In static pools, we must choose multiple concrete node configurations up front—i.e., for replicas: 10, we select 10 NodeClaims matching the requirement set.
+If the requirements allow multiple combinations:
+- Karpenter selects the optimal combination based on cost, availability, and zone balancing
+- This selection is done once at provisioning time (unlike dynamic pools, where evaluation occurs per provisioning event)
+
+In static provisioning, the NodeClaim requirements are directly derived from the NodeClaimTemplate on the NodePool. These are evaluated once per NodeClaim at creation, meaning the selection is based solely on what the template allows.
+As a result, even though all NodeClaims come from the same static NodePool, they may still result in different instance types (shapes/flavors), depending on availability, since that decision happens during cloud-provider Create() call.
+
+#### Zonal Distribution
+Static NodePools do not currently support topology-aware spreading in the same way that dynamic NodePools rely on pod-driven scheduling. Initially, we explored automatically spreading static NodeClaims across zones when the topology.kubernetes.io/zone requirement includes multiple values, for example:
+
+```yaml
+- key: topology.kubernetes.io/zone
+  operator: In
+  values: ["zone-2a", "zone-2b", "zone-2c"]
+```
+
+However, this introduces implicit behavior that conflicts with existing Karpenter semantics. In current behavior, absence of a requirement is treated as unconstrained, and specifying zones in requirements is interpreted as a hard constraint. Introducing automatic spreading implicitly redefines these semantics, which could have long-term consequences for other requirement types (e.g., capacity type).
+Given these concerns, we’re punting zonal spreading to a future enhancement. For now, we will continue to treat the requirements field as the single source of truth for topology decisions. Users who want to spread nodes across zones can do so explicitly by:
+- Specifying multiple zones in the topology.kubernetes.io/zone requirement, or
+- Creating multiple static NodePools, each pinned to a specific AZ.
+
+We plan to explore a nodeTopologySpread field in the future to offer users a more flexible and declarative way to express zone-spreading behavior, similar to podTopologySpreadConstraints.
```

Comment:
I wouldn't prescribe a specific field, I'd just say we're going to explore a method to decoratively express intention to spread nodes across zones.